chromadb/test/conftest.py

```diff
@@ -183,26 +279,62 @@ def sqlite_persistent() -> Generator[System, None, None]:
 
 
 def system_fixtures() -> List[Callable[[], Generator[System, None, None]]]:
-    fixtures = [fastapi, fastapi_persistent, sqlite, sqlite_persistent]
+    fixtures = [
```

Comment:
Do we really need to add the auth'ed server here? It increases test times by quite a bit because many downstream tests use it.

Reply:
I agree, i added it there as shortcut of running both regression and coverage for all possible auth API calls that need auth but I can remove it and instead create tests for each endpoint separately.

Reply:
Cool, thanks.